src/ape/managers/project.py

```diff
@@ -1515,7 +1516,51 @@ def from_manifest(
         Returns:
             :class:`~ape.managers.project.ProjectManifest`
         """
-        return Project.from_manifest(manifest, config_override=config_override)
+        config_override = config_override or {}
+        manifest = _load_manifest(manifest) if isinstance(manifest, (Path, str)) else manifest
+        return Project(manifest, config_override=config_override)
+
+    @classmethod
+    def from_python_library(
```

Comment:
in addition to this, we could have a `PythonDependency(DepdendencyAPI)` where you config like:

```
dependencies:
  python: snekmate
```

somehow need this to happen automatically though....